### PR TITLE
<WIP> Add social buttons

### PR DESCRIPTION
**Pull request by uLan08**
Changes: Add social buttons. Tweetting would say `My top used language is JavaScript https://githublangs.herokuapp.com?username=uLan08 #github #JavaScript`. Unfortunately, it seems that facebook does not support automatic filling of the caption. I also had to pass the variable `goodAt` to the view.

Screenshots for the change:
screenshot from 2017-10-13 16-38-45

I have a slight bug, that I haven't fixed yet. I was hoping you could help me fix it.
The bug is when I type again a username the social buttons doesn't disappear even though they are inside the main `div`
screenshot from 2017-10-13 16-39-11

### CODE REVIEW

**Comment by wildan3105** [comment]
Okay I'll check it first @uLan08

**Comment by uLan08** [comment]
I found out that the social buttons overlap the text `For users with a lot of repo, only shown top 20 languages`. I'll try to see if I can fix this. This really tests my frontend skills as I am really more of a backend guy :sweat:

File: views/partials/footer.handlebars [hunk 1]
```diff
@@ -7,3 +7,30 @@
```
Comment (wildan3105) [inline views/partials/footer.handlebars]: please placed in `lib/public/js/social.js` instead in `footer.handlebars`

**Comment by wildan3105** [comment]
Same here @uLan08 :smile: I'm also a backend guy trying to understand full-stack :laughing:

**Comment by wildan3105** [comment]
And also please include 
- [ ] screenshot of your fixes
- [ ] screenshot of twitter tweet
- [ ] screenshot of facebook share

**Comment by wildan3105** [comment]
One last thing, could you make `share` and `tweet` button side by side and not top-down?

**Comment by uLan08** [comment]
Screenshots 

 screenshot of your fixes
screenshot from 2017-10-13 17-48-24

 screenshot of twitter tweet
screenshot from 2017-10-13 17-49-18

 screenshot of facebook share
screenshot from 2017-10-13 17-49-33

**Comment by wildan3105** [comment]
Thanks !
What about this : 
> One last thing, could you make share and tweet button side by side and not top-down?

**Comment by uLan08** [comment]
I'll be doing that now. I just uploaded the screenshots first.

**Comment by uLan08** [comment]
screenshot from 2017-10-13 18-25-52

Everything's good so far. The only thing left is the one where it doesn't disappear when it's reloading

**Comment by wildan3105** [comment]
Okay, I'll check it, thanks for the work so far! @uLan08

**Comment by uLan08** [comment]
Changed the word `top` to `most` on tweet. 
screenshot from 2017-10-13 18-32-16

**Comment by wildan3105** [comment]
Hi @uLan08, could you resolve the conflicts? And I tried yesterday the `tweet` button worked but facebook `share` had a problem about appId? Thanks! Looking for your change :)

**Comment by uLan08** [comment]
Alright. I'll get to it when I can get on my computer.

**Comment by uLan08** [comment]
screenshot from 2017-10-16 11-50-13

**Comment by wildan3105** [comment]
Excellent work! Thanks @uLan08 ! :100:

**Comment by uLan08** [comment]
Thanks as well!

**Pull request merged**